ide/css.editor/src/org/netbeans/modules/css/editor/csl/CssCompletion.java

```diff
@@ -109,7 +109,7 @@ public class CssCompletion implements CodeCompletionHandler {
     @Override
     public CodeCompletionResult complete(CodeCompletionContext context) {
 
-        final List<CompletionProposal> completionProposals = new ArrayList<>();
+        List<CompletionProposal> completionProposals = new ArrayList<>();
 
         CssParserResult info = (CssParserResult) context.getParserResult();
```

Comment:
info variable seems redundant

Reply:
a) When debugging it can be helpful to have the variable as local
b) This was not modified in this PR, so won't be touched as no improvment will be the result.

Reply:
sure